openjdk/mmtkHeap.cpp

```diff
@@ -83,13 +83,25 @@ jint MMTkHeap::initialize() {
   const size_t max_heap_size = collector_policy()->max_heap_byte_size();
   //  printf("policy max heap size %zu, min heap size %zu\n", heap_size, collector_policy()->min_heap_byte_size());
 
-  // Set options
-  mmtk_builder_set_threads(ParallelGCThreads);
-  mmtk_builder_set_transparent_hugepages(UseTransparentHugePages);
+  // Read MMTk options from environment variables (such as `MMTK_THREADS`).
+  // i.e. Environment variable options (priority 3) override OpenJDK's default options (priority 2).
```

Comment:
This priority should be documented somewhere. Either in the `README` or user guide.

Reply:
Done.  I documented the priorities of different option sources in openjdk/thirdPartyHeapArguments.cpp